JapaneseFilter/sections/antiadblock.txt

```diff
@@ -11,6 +11,9 @@
 ! https://jbbs.shitaraba.net/bbs/read.cgi/internet/25463/1652115193/524
 blog-and-destroy.com#%#//scriptlet('prevent-addEventListener', 'load', 'adsCount')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/129121
+! https://github.com/AdguardTeam/AdguardFilters/issues/130283
+@@||manga1001.vip^$cookie
```

Comment:
Do we really need this rule (`@@||manga1001.vip^$cookie`)?
It seems that without this rule website works correctly.

Additional/Alternative solution could be something like:
```adblock
manga1001.vip#%#//scriptlet("abort-current-inline-script", "document.querySelector", "_0x")
```
or just:
```adblock
manga1001.vip#@#.c-ads 
manga1001.vip#@#.body-top-ads
```
but I guess that `$generichide` in this case is okay.